tests/integration_tests/util.py

```diff
@@ -240,14 +240,14 @@ def _verify_clean_boot(
 
         # check for required warnings
         for expected in require_warnings:
-            if expected in current_warning:
+            if re.compile(expected).search(current_warning):
```

Comment:
Do all current tests pass like this? 

Suddenly turning exact match string checks into regex checks seems like a bad idea. 

Regardless, we already have a way to accomplish this that doesn't require changing the behavior of existing tests. Why not just make the checks series-specific?